### ISSUE TITLE
yolov4 -d 出错

### ISSUE THREAD

**Issue by qingchunnianshao** [wang-xinyu/tensorrtx#328]
您好，下载配置了yolov4 代码在VS2015中配置运行环境 yolov4.cpp中
addBatchNorm2d(network, weightMap, *conv1->getOutput(0), "module_list." + std::to_string(linx) + ".BatchNorm2d", 1e-4);
该句话提示abort() has been called.
![image](https://user-images.githubusercontent.com/16200499/101712360-53e42000-3ad0-11eb-8bd3-923932a7a145.png)

环境：
1.Win7
2.TensorRT-7.0.0.11
3.CUDA10.2
4.Cudnn-10.2-windows-x64-v7.6.5.32

gen_wts.py 生成wts。文件代码如下
import struct
import sys
from models1 import *
from utils.torch_utils import *
import utils.torch_utils
model = Darknet('cfg/yolov4.cfg', (608, 608))
weights = sys.argv[1]
device = select_device('0')
#device = torch_utils.select_device('0')
if weights.endswith('.pt'):  # pytorch format
    model.load_state_dict(torch.load(weights, map_location=device)['model'])
else:  # darknet format
    Darknet.load_darknet_weights(model, weights)

f = open('yolov4.wts', 'w')
f.write('{}\n'.format(len(model.state_dict().keys())))
for k, v in model.state_dict().items():
    vr = v.reshape(-1).cpu().numpy()
    f.write('{} {} '.format(k, len(vr)))
    for vv in vr:
        f.write(' ')
        f.write(struct.pack('>f',float(vv)).hex())
    f.write('\n')

请问是哪里出现问题了呢？

**Comment by qingchunnianshao** [comment]
由于上面问题还没定位问题所在，尝试运行最简单的lenet, lenet -s 能够正常运行，但是 lenet -d
    ICudaEngine* engine = runtime->deserializeCudaEngine(trtModelStream, size, nullptr); engine 一直返回为空，能否指导下是什么问题

**Comment by wang-xinyu** [comment]
windows要注意运行路径
